tox.ini

```diff
@@ -0,0 +1,93 @@
+[tox]
+envlist =
+    py{38,39,310,311}-test{,-oldestdeps,-devdeps,-predeps}
+    codestyle
+    twine
+    bandit
+requires =
+    setuptools >= 30.3.0
```

Comment:
No need for `setuptools_scm` in `tox.ini`?

Reply:
`astropy` does not list it, and it is already declared in pyproject.toml . But if you want it listed, doesn't hurt. 🤷

Reply:
Only if an accurate version number is needed in the testing?

Reply:
Maybe. I actually don't fully understand this section. I was just copy pasting from a setup I know will work. Do you want me to remove it and see?

Reply:
I guess if tox does not bypass the `setup.cfg` then it shouldn't be necessary ...?

Reply:
> Do you want me to remove it and see?

I think we need it in `setup.cfg` (and likely `pyproject.toml`). I just don't know how `tox` interacts with those, but presumably uses pip to install things so I suppose that is good enough.  So no need to change anything here, I guess.

Reply:
I removed it. I am curious now.